de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/navigation/proxy/FolderRpcProxyImpl.java

```diff
@@ -65,15 +66,17 @@ public void onSuccess(List<DiskResourceQueryTemplate> result) {
         final IsMaskable maskable;
         private final NavigationView.Presenter.Appearance appearance;
         final SearchServiceFacade searchSvc;
+        UserInfo userInfo;
 
         public RootFolderCallback(final SearchServiceFacade searchService,
                                   final AsyncCallback<List<Folder>> callback,
-                                  final IplantAnnouncer announcer,
+                                  final IplantAnnouncer announcer, UserInfo userInfo,
```

Comment:
All other params in this constructor are on their own line, so this `UserInfo` param should also be on its own line.